packages/repository/src/__tests__/acceptance/has-many-through.relation.acceptance.ts

```diff
@@ -0,0 +1,247 @@
+// Copyright IBM Corp. 2019. All Rights Reserved.
+// Node module: @loopback/repository
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import * as _ from 'lodash';
+import {Application} from '@loopback/core';
+import {expect} from '@loopback/testlab';
+import {
+  ApplicationWithRepositories,
+  juggler,
+  repository,
+  RepositoryMixin,
+} from '../..';
+import {Order, Seller} from '../fixtures/models';
+import {
+  CustomerRepository,
+  OrderRepository,
+  SellerRepository,
+} from '../fixtures/repositories';
+
+describe('HasManyThrough relation', () => {
```

Comment:
We are in the process of migrating relation acceptance tests to `packages/repository-tests` so that we can run the tests against different databases (`memory`, `mysql` and `mongodb` ATM, more databases will come in the future) - see https://github.com/strongloop/loopback-next/issues/3442 and https://github.com/strongloop/loopback-next/pull/3538.

Can you please add these new tests to `repository-tests` from the beginning, so that we don't have to move them later?